File: io1.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	fmt.Printf("Some Input Please? ")
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Scan()
	line := scanner.Text()
	fmt.Printf("Echo of ->%s<-\n", line)
}

File: io1.js
var readline = require('readline');

var rl = readline.createInterface({
	input: process.stdin,
	output: process.stdout,
	terminal: true
});

process.stdout.write ( "Some Input Please? " );
rl.on('line', function (line) {
	console.log ( "Echo	of ->" + line + "<-" );
	process.exit();
});
